src/Emulator/DialogueBox/DialogueBox.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useRef, useEffect } from 'react';
```

Comment:
There's a fairly large gap between the right side of the panel and the left side of the image/configure panel that wasn't there before. Can you fix this?

![image](https://user-images.githubusercontent.com/22752497/102034553-e52eed00-3d83-11eb-86f9-7015c05571ff.png)

Whenever the screen get's a little smaller the right side of the panel get cropped off.

![image](https://user-images.githubusercontent.com/22752497/102034647-2e7f3c80-3d84-11eb-9a3d-1f8f246e4d56.png)